docs/contrib/build.rst

```diff
@@ -103,6 +103,10 @@ whereas the `release` mode performs additional optimizations and takes longer
 to compile. The `release-fast` mode builds faster, performs less optimizations,
 but may perform better than `release`.
 
+The `optimize` setting is controlled via the `SCALANATIVE_OPTIMISE` environment
```

Comment:
One that was forgotten:
```suggestion
The `optimize` setting is controlled via the `SCALANATIVE_OPTIMIZE` environment
```

Reply:
Ah, I did case sensitive search and replace.